docs/content/docs/dev/table/sql/create.md

```diff
@@ -554,7 +554,14 @@ INSERT INTO my_ctas_table SELECT id, name, age FROM source_table WHERE mod(id, 1
 * Does not support creating partitioned table yet.
 * Does not support specifying primary key constraints yet.
 
-**Note** The target table created by CTAS is non-atomic currently, the table won't be dropped automatically if occur errors while inserting data into the table.
+**Note** By default, the target table created by CTAS is non-atomic, the table won't be dropped automatically if occur errors while inserting data into the table.
+
+#### Atomicity
+
+If you want to implement atomicity CTAS, then you need to fulfill the following conditions at the same time:
+* Set option `table.rtas-ctas.atomicity-enabled` to `true`.
```

Comment:
Please make the following content be the first point:
```
* The sink has implemented the atomicity semantics for CTAS.  You may refer to the corresponding doc for the sink to know the atomicity semantics is available or not.  For devs who want to implement the atomicity semantics,  please refer to doc [SupportsStaging](link).
```
Then add a part for the interface `SupportsStaging` in doc table/sourcessinks/#sink-abilities just like other sink abilities.